people/jenna/coffeetea.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tea v Coffee </title>
+</head>
+<body>
+
+<!-- Testing page to try out if and else statements-->
+
+
+<h1 style="text-align: center"> Tea & Coffee </h1>
+<br> <center><img src="https://uploads.disquscdn.com/images/7b7eb86cc5695faa31ee4c015f42cba1f3d9a9e719c2f020dd14b64234783741.jpg"></center>
+
+
+
+<center>
+<h2 style="text-align: center"> Which type of drink is right for you? </style> </h1>
+
+<p>How do you feel?</p>
+
+<select id="feeling" onchange="myFunction()">
+  <option value="none">
+  <option value="Pumped">Pumped
+  <option value="Drowsy">Drowsy
+ 
+</select>
+
+
+<br><br>
+<p>Strength?</p>
+
+<select id="type" onchange="myFunction()">
+  <option value="none">
+  <option value="Refreshing">Refreshing
+  <option value="Strong">Strong
+</select>
+
+<p>Here is your drink....</p>
+
+<p id="demo"></p>
+
+<script>
+function myFunction() {
```

Comment:
Make sure your function names are descriptive of what they do! Perhaps `makeSuggestion()`?